frontend/app/work_packages/view_models/common-relations-handler.js

```diff
@@ -26,7 +26,7 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
-module.exports = function($timeout, WorkPackageService, ApiHelper, PathHelper, MAX_AUTOCOMPLETER_ADDITION_ITERATIONS) {
+module.exports = function($timeout, WorkPackageService, ApiNotificationsService, PathHelper, MAX_AUTOCOMPLETER_ADDITION_ITERATIONS) {
```

Comment:
Line is too long.<br>'MAX_AUTOCOMPLETER_ADDITION_ITERATIONS' is defined but never used.<br>'PathHelper' is defined but never used.